frontend/src/Component/DashViewApplication.tsx

```diff
@@ -6,31 +6,29 @@ import "/src/App.css"
 import {JobApplicationTracker} from "../types/JobApplicationTracker.ts";
 import JobApplicationCard from "./JobApplicationCard.tsx";
 
-export function DashViewApplication(){
-    const [odata,setOdata]=useState<JobApplicationTracker[]>([])
-    function getApplications(){
-        axios.get<JobApplicationTracker[]>("/api/JobApplication").then(
-            (response)=>{
+export function DashViewApplication() {
+    const [odata, setOdata] = useState<JobApplicationTracker[]>([])
```

Comment:
`odata` could be better named. I don't understand what it holds. A better name for the state variable could be `applicationTrackerData`